package.json

```diff
@@ -63,5 +63,9 @@
     "ts-node": "^10.9.1",
     "typescript": "<5.4.0",
     "yaml": "^2.3.4"
+  },
+  "devDependencies": {
+    "dotenv": "^16.4.5",
+    "eslint-plugin-yaml": "^1.0.3"
```

Comment:
This seems out of place. This project doesn't release a JavaScript library, so move these into `dependencies` to be consistent.